cpu/nrf52/periph/i2c.c

```diff
@@ -115,6 +115,31 @@ void i2c_init(i2c_t dev)
     bus(dev)->ENABLE = TWIM_ENABLE_ENABLE_Enabled;
 }
 
+#ifdef MODULE_PERIPH_I2C_RECONFIGURE
+void i2c_init_pins(i2c_t dev)
+{
+    assert(dev < I2C_NUMOF);
+
+    gpio_init(i2c_config[dev].scl, GPIO_IN_OD_PU);
+    gpio_init(i2c_config[dev].sda, GPIO_IN_OD_PU);
+    bus(dev)->PSEL.SCL = i2c_config[dev].scl;
+    bus(dev)->PSEL.SDA = i2c_config[dev].sda;
+
+    bus(dev)->ENABLE = TWIM_ENABLE_ENABLE_Enabled;
+
+    mutex_unlock(&locks[dev]);
+}
+
+void i2c_deinit_pins(i2c_t dev)
+{
+    assert(dev < I2C_NUMOF);
+
+    mutex_lock(&locks[dev]);
+    bus(dev)->ENABLE = TWIM_ENABLE_ENABLE_Disabled;
+
```

Comment:
```suggestion
```

Reply:
Thanks for the quick feedback. 
Does that mean, you want me to remove this?

Reply:
just the extra newline at the end